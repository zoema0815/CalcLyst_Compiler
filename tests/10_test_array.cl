int main() {
    int[2] int_list;
    float[2] flt_list;
    int_list[0] = 4;
    int_list[1] = 2;
    flt_list[0] = 2.3;
    flt_list[1] = 4.67;
    printInt(int_list[0]);
    printInt(int_list[1]);
    printInt(int_list[0] + int_list[1]);
    printInt(int_list[0] - int_list[1]);
    printInt(int_list[0] * int_list[1]);
    printInt(int_list[0] / int_list[1]);
    printFloat(flt_list[0]);
    printFloat(flt_list[1]);
    printFloat(flt_list[0] + flt_list[1]);
    printFloat(flt_list[0] + flt_list[1]);
    printFloat(flt_list[0] * flt_list[1]);
    printFloat(flt_list[0] / flt_list[1]);
    return 0;
}