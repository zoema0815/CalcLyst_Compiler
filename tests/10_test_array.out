4
2
6
2
8
2
2.3
4.67
6.97
6.97
10.741
0.492505
